spring-beans/src/main/java/org/springframework/beans/BeanUtils.java

```diff
@@ -677,7 +679,8 @@ public static void copyProperties(Object source, Object target, String... ignore
 	}
 
 	/**
-	 * Copy the property values of the given source bean into the given target bean.
+	 * Copy the property values of the given source bean into the given target bean
+	 * and ignored if 
```

Comment:
It looks like this change was made in error.

Reply:
Okay I see